components/src/preact/mutationFilter/mutation-filter.tsx

```diff
@@ -115,16 +128,24 @@ function MutationFilterInner({ initialValue }: MutationFilterInnerProps) {
             const values = newInputValue.split(',').map((value) => {
                 return { value, parsedValue: parseAndValidateMutation(value.trim(), referenceGenome) };
             });
-            const validEntries = values.map((value) => value.parsedValue).filter((value) => value !== null);
-            const invalidInput = values
-                .filter((value) => value.parsedValue === null)
-                .map((value) => value.value.trim())
-                .join(',');
+
+            const validEntries: MutationFilterItem[] = [];
+            const rejected: string[] = [];
+
+            for (const v of values) {
+                if (!v.parsedValue) {
```

Comment:
```suggestion
                if (v.parsedValue === null) {
```
Let's please be explicit to avoid subtle edge cases with things that are falsy.